universal-application-tool-0.0.1/conf/evolutions/default/2.sql

```diff
@@ -0,0 +1,13 @@
+# --- Sample dataset
```

Comment:
Fantastic!  Can you elaborate a little on how these are created and how to write more of these?  I assume it's the structured comments and their location under conf/evolutions/default that make them work, but if there's more to it it'd be great to have that written down.

Reply:
I followed the setup here https://github.com/playframework/play-samples/blob/2.8.x/play-java-ebean-example/conf/evolutions/default/2.sql.

Reading from the [documentation](https://www.playframework.com/documentation/2.8.x/Evolutions#Evolutions-scripts), the scripts for each database should locate in `conf/evolutions/{database name}` and numbered in name.

The relevant comments are `-- !Ups` and `-- !Downs`. The rest seems to be regular comments.

Reply:
Cool, works for me!  Can you add a section to `README.md` about this?  Some questions to answer:

- is the database created newly every time the app starts up?
- how do we add new schema elements / new tables / etc?
- how do we ensure that the db is up to date?
- if we need to undo a db schema change we're working on in dev, how do we do that?